src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementTimeZoneTest.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.services;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.opentest4j.AssertionFailedError;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.ReportableTaskResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskHistoryResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskResource;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.UUID;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.CFTTaskState.UNASSIGNED;
+
+@ActiveProfiles("replica")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+@Slf4j
+class TaskManagementTimeZoneTest extends ReplicaBaseTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"UTC", "BST"})
+    void when_timezone_changes_all_timestamp_attributes_should_behave_consistently(String timeZone) {
+
+        TaskResource taskResource;
+        log.info("TimeZone ({})", timeZone);
+
+        if ("UTC".equals(timeZone)) {
+            taskResource = createAndSaveTask(OffsetDateTime.now(ZoneOffset.UTC));
+        } else {
+            taskResource = createAndSaveTask(OffsetDateTime.now(ZoneOffset.of("+01:00")));
+        }
+
+
+        await().ignoreException(AssertionFailedError.class)
+            .pollInterval(1, SECONDS)
+            .atMost(10, SECONDS)
+            .until(
+                () -> {
+                    List<TaskHistoryResource> taskHistoryResourceList
+                        = miReportingServiceForTest.findByTaskId(taskResource.getTaskId());
+
+                    assertFalse(taskHistoryResourceList.isEmpty());
+
+                    assertTrue(taskResource.getLastUpdatedTimestamp().truncatedTo(ChronoUnit.SECONDS)
+                                   .isEqual(taskHistoryResourceList.get(0).getUpdated()
+                                                .truncatedTo(ChronoUnit.SECONDS)));
```

Comment:
Should the DBs be storing everything in UTC and therefore be represented with no offset?  Do we need to check that too?